docs/manual/manual_throttle.rst

```diff
@@ -0,0 +1,346 @@
+on the DroneCAN bus, with each one intended for one module. These raw commands are sent as an integer value from -8192 to 8191. 
+
+To convert these raw commands to a throttle command, simply divide the value by 8192. For example, a raw command of -8192 corresponds to a 
+throttle command of -100%, and a raw command of 4096 corresponds to a throttle command of 50%.
+
+Hobby Protocols
+****************
+The :ref:`manual_hobby` section covers in more detail how each hobby protocol communicates with Vertiq modules. Generally, each type of hobby protocol 
+sends a command that encodes a number from 0.0 to 1.0. For example, :ref:`Standard PWM <hobby_standard_pwm>` sends a pulse from 1000 microseconds to 2000 microseconds, where 1000 microseconds 
+corresponds to a 0.0 and 2000 microseconds corresponds to a 1.0.
+
+Similarly to the :ref:`throttle_iquart_ref` throttle command, these commands only encode a range from 0.0 to 1.0, while a throttle command should be able to cover a range from -100% 
+to 100%. The full range of -100% to 100% throttle commands can be sent by hobby protocols, but not directly as a -1 to 1 value. Instead, depending on the 
+configuration of your module the 0.0 to 1.0 value that hobby protocols send will be re-mapped to the full -100% to 100% range. For more details on how this mapping 
+works and how to configure it, see the :ref:`throttle_mapping` section.
+
+.. _throttle_mode_maximums_directions:
+
+Mode, Maximums and Direction
+==============================
+To fully understand how a Vertiq module will interpret a throttle command, there are several key configurations on a Vertiq module that need to be covered. 
+The definition of a throttle command is covered in the :ref:`throttle_def` section, 
+
+For a tutorial walking through setting these parameters up in more detail, see the :ref:`control_center_tutorial` and :ref:`hobby_fc_tutorial` tutorials.
+
+.. note:: The descriptions in this section of how a module will spin in response to a throttle command assume that the zero spin throttle is left at its default setting, where a 0% throttle 
+    command is the zero spin throttle. The exact rate at which a module spins in response to a throttle command may differ from the descriptions in the following sections if the 
+    zero spin throttle has been configured differently. For more information on how the module's response to throttle commands will change, see the :ref:`manual_zero_spin` section.
+    These descriptions also assume that the module has already :ref:`armed <manual_advanced_arming>` before spinning.
+
+.. _throttle_mode:
+
+Mode
+*****
+Every Vertiq module should be configured to operate in a certain “mode”. This mode determines how the module will interpret throttle commands it receives and 
+how it will attempt to control itself. There are three possible modes, covered in the sections below.

```

Comment:
either no comma in the last sentence, or make it "There are three possible modes, each overed in the sections below"